op-program/client/interop/interop.go

```diff
@@ -63,6 +63,9 @@ func stateTransition(logger log.Logger, bootInfo *boot.BootInfoInterop, l1Preima
 	if err != nil {
 		return common.Hash{}, err
 	}
+	if superRoot.Timestamp >= bootInfo.GameTimestamp {
```

Comment:
Don't we want strict equality here?

If the boot info claim is < the super root / prestate timestamp, that's 100% wrong (the program enforces that the chain must be extended by 1 block), and we should reject it yeah?

Reply:
I mean if you agree to a prestate that has already gone past the correct place to stop you're not the honest actor.  I'm not sure there's anything we can really do that's sensible there.  So for the honest actor >= or == is the same because you'll never have a prestate that's >.  For everything else it feels like a toss up and I figured not attempting to do a derivation that doesn't make sense seemed better.  Maybe you could transfer to invalid but I worry about giving that two meanings - though again, probably doesn't matter since the honest actor can't hit that case.

Reply:
I should add a comment to that effect though.  And I'm not against changing it if we want to but I'm not sure there's a "good" correct hash to choose if you've already gone past the final timestamp.

Reply:
Added a comment and switched it to panic if the agreed timestamp is after the game timestamp. The game is going to go wrong whatever result we pick so may as well get a helpful message...